packages/base-domains/base-domains.base/opam

```diff
@@ -6,7 +6,6 @@ Domains-based parallelism distributed with the Multicore OCaml compiler"
 depends: [
   "ocaml" {>= "5.0"} |
   "ocaml-variants" {
-    = "4.14.0+domains" |
     = "4.12.0+domains+effects" |
```

Comment:
```suggestion
    = "4.14.0+domains" |
    = "4.12.0+domains+effects" |
```

Reply:
We can't remove this since, although it is deprecated, it still exists in the multicore overlay and removing it could break things. It is harmless anyway